packages/server/src/arpa_reporter/services/validation-rules.js

```diff
@@ -1,12 +1,16 @@
 
 const srcRules = require('../lib/templateRules.json')
 
+function isStr(val) {
+  return val && typeof val === 'string'
+}
```

Comment:
I think it's fine either way, but just wanted to mention that we have `lodash` available for use in this package, which also provides an [`_.isString(value)` function](https://lodash.com/docs/#isString). I think the (sole?) advantage in this case would be having one less thing unit to track in coverage metrics 🤷